### DISCUSSION TITLE
Release Date of 1.5.5

### DISCUSSION THREAD

**Discussion by cansik** [bytedeco/javacpp#450] [General]
Is there already a fixed release date of `1.5.5` or do you have a roadmap?
Would it make sense to have a more or less fixed schedule of the releases?

This questions does not only relate to javacpp but also the javacpp-presets.

**Comment by saudet** [comment]
I did once every 3 months for a while, but no one cared, so I don't know.
Why didn't you care until now? What changed?

**Comment by saudet** [comment]
If what you'd like to have is just the new version of librealsense2, we can do that pretty easily.
I need to test the release pipeline with GitHub Actions anyway, so that would be good practice.
It's not worth making a broken release though, so please make sure that everything is working correctly with the snapshots.

**Comment by cansik** [comment]
Well I was just interested, but yes, it is also because of the librealsense2 library, which could be used to control the most recent cameras. Snapshots worked so far on MacOS, I could not test yet other operating systems.

**Comment by saudet** [comment]
Ok, I've released librealsense2 2.40.0-1.5.4. Enjoy!
https://repo1.maven.org/maven2/org/bytedeco/librealsense2/2.40.0-1.5.4/
